turbinia/config/turbinia_config.py

```diff
@@ -82,24 +81,23 @@
 # GCP AND GCE #
 ###############
 
-PROJECT = None
-ZONE = None
-INSTANCE = None
-DEVICE_NAME = None
-SCRATCH_PATH = None
+PROJECT = 'None'
+ZONE = 'None'
+INSTANCE = 'None'
+DEVICE_NAME = 'None'
+SCRATCH_PATH = 'None'
 # GCS bucket that has Turbinia-specific scripts and can be used to store logs.
-BUCKET_NAME = None
+BUCKET_NAME = 'None'
 PSQ_TOPIC = 'turbinia-psq'
 # TODO(beamcodeup): Per https://github.com/google/turbinia/issues/172, Cloud
 # Functions are only available on us-central1. Thus, hardcoding this for now.
 # Fix this when CF starts supporting more regions.
-TURBINIA_REGION = 'us-central1'
-
+TURBINIA_REGION = 'None'
 
 # Topic Turbinia will listen on for new Artifact events. This is also used as
 # the Turbinia instance/namespace as it is a unique string per Turbinia
 # instance and Cloud Project.
-PUBSUB_TOPIC = 'turbinia-pubsub'
+PUBSUB_TOPIC = 'None'
```

Comment:
Sorry, I should have commented on this one specifically too (as far as keeping the existing default value).   Could we keep this one as well?  I can always put it back in a separate PR too.

Reply:
Done.